packages/server/graphql/private/typeDefs/Mutation.graphql

```diff
@@ -14,10 +14,54 @@ type Mutation {
     description: String!
 
     """
-    The expiration date of the feature flag (optional)
+    The expiration date of the feature flag
     """
     expiresAt: DateTime!
   ): AddFeatureFlagPayload!
+
+  """
+  Apply a feature flag to specified subjects (users, teams, or organizations)
+  """
+  applyFeatureFlag(
+    """
+    The name of the feature flag to apply
+    """
+    flagName: String!
```

Comment:
`DeleteFeatureFlagSuccess` returns the type `featureFlag: FeatureFlagNew!` (after the refactor it'll be `FeatureFlag` instead of `FeatureFlagNew`), which is the `FeatureFlag` object, not an enum

Reply:
Sorry, missed this type because I only checked this PR. All good then.